conntrack/conntrack.go

```diff
@@ -256,6 +256,7 @@ func (h *Handles) SendMessage(hdr *syscall.NlMsghdr, data []byte) error {
 
 func (h *Handles) sendMessage(hdr *syscall.NlMsghdr, data []byte) error {
 	sh, err := h.open()
+	defer sh.close()
```

Comment:
This should be done after checking the error, not before. This will panic otherwise if there is an error.